pkg/volume/csi/proto/csi/BUILD

```diff
@@ -0,0 +1,27 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
```

Comment:
This file should no longer exist, since the proto is vendored.

Reply:
Not sure whay `hack/update-bazel.sh` keeps creating that BUILD file.  There's nothing else in that dir.  will check again.